pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -73,7 +72,10 @@ class LightningTensor(Device):
 
             ``max_bond_dim`` (int): Maximum bond dimension for the MPS simulator.
                 It corresponds to the number of Schmidt coefficients retained at the end of the SVD algorithm when applying gates. Default is ``None``.
-            ``cutoff`` (float): Truncation threshold for the Schmidt coefficients in a MPS simulator. Default is ``1e-16``.
+            ``cutoff`` (float): Truncation threshold for the Schmidt coefficients in a MPS simulator. Default is ``np.finfo(c_dtype).eps``.
+            ``contract`` (str): The contraction method for applying gates. It can be either ``auto-mps`` or ``nonlocal``.
+                ``nonlocal`` turns each gate into a MPO and applies it directly to the MPS, while ``auto-mps`` swaps nonlocal qubits in 2-qubit gates to be next
```

Comment:
Do we want to update the docstring here since multiple qubits gates are supported as well?

Reply:
In which sense? Maybe I misunderstood, but the fact that multiple qubits gates are supported is communicated in the list of supported gates/observables.